x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/windows_suspicious_pdf_reader.json

```diff
@@ -1,35 +0,0 @@
-{
```

Comment:
why'd this one disappear?

Reply:
was it a rename that github didn't pick up as a diff?

Reply:
apparently git didn't detect the move to execution_suspicious_pdf_reader.json